### PR TITLE
Add RunInference API to CHANGES.md

### PR DESCRIPTION
**Pull request by yeandy**
Adding a note on the new RunInference API to CHANGES.md

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] [**Choose reviewer(s)**](https://beam.apache.org/contribute/#make-your-change) and mention them in a comment (`R: @username`).
 - [ ] Add a link to the appropriate issue in your description, if applicable. This will automatically link the pull request to the issue.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/#make-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI.

### CODE REVIEW

**Comment by yeandy** [comment]
R: @ryanthompson591 @AnandInguva

**Comment by yeandy** [comment]
R: @TheNeuralBit @tvalentyn

File: CHANGES.md [hunk 1]
```diff
@@ -53,8 +53,8 @@
 
 ## Highlights
 
-* New highly anticipated feature X added to Python SDK ([BEAM-X](https://issues.apache.org/jira/browse/BEAM-X)).
-* New highly anticipated feature Y added to Java SDK ([BEAM-Y](https://issues.apache.org/jira/browse/BEAM-Y)).
+* Initial preview of the [RunInference](https://s.apache.org/inference-sklearn-pytorch) API added to Python SDK ([BEAM-13970](https://issues.apache.org/jira/browse/BEAM-13970)).
```
Comment (TheNeuralBit) [inline CHANGES.md:56]: Do we want to call it a preview here?

Comment (yeandy) [reply CHANGES.md:56]: Bad wording on my part. How does this sound: "Added the new RunInference API to the Python SDK. Users can now make inferences with PyTorch and Scikit-learn models!". Open to suggestions/changes.

Comment (TheNeuralBit) [reply CHANGES.md:56]: Yeah that sounds better to me. You might also check with @rezarokni to see how he wants to message it. (might need to reach out to him directly, I don't know that he'll see a ping from GitHub).

Comment (rezarokni) [reply CHANGES.md:56]: How about "Added [RunInference](https://s.apache.org/inference-sklearn-pytorch) API, a framework agnostic transform for inference. With this release PyTorch(<config link>)  and Scikit-learn (<config link>) are supported by the transform.

**Comment by yeandy** [comment]
@rezarokni PTAL

**Review by rezarokni — APPROVED** [review]

**Pull request merged**